packages/compass-saved-aggregations-queries/src/stores/open-item.ts

```diff
@@ -47,16 +64,23 @@ export enum ActionTypes {
 
 type OpenModalAction = {
   type: ActionTypes.OpenModal;
-  selectedItem: Item;
+  modal: OpenedModal;
+  selectedItem?: Item;
+  connections?: State['connections'];
```

Comment:
why is this optional? when would we not send the connections by opening (and use the `state.connections` instead as written below?)

Reply:
When we open the no-active-connections modal that's when we will not send any connections.